examples/prediction/xgboost-high-level-apis.ipynb

```diff
@@ -245,16 +245,42 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "(train_X, train_y), (test_X, test_y) = read_input('ames_dataset/train.csv')\n",
-    "endpoint.predict(test_X)\n"
+    "import requests\n",
```

Comment:
I am assuming we'll have a follow up PR to fix this.

Reply:
yes, I assume @r2d4 is working on this.